nix/default.nix

```diff
@@ -19,6 +19,9 @@
   # Autogenerated Nix from `spago2nix generate`
 , spagoPackages ? "${src}/spago-packages.nix"
   # Configuration that will be used to generate a `devShell` for the project
+  # shell.
+  # `shell.packageLockOnly`: When set to true, the `node_modules` folder will
+  # not be built when `npm` is used inside the shell.
 , shell ? { }
 , ...
 }:
```

Comment:
Is this a good place to place the documentation, or should I move it to the `shellFor` definition? @ngua

Reply:
No, I'd move it to `shellFor`

Reply:
Seeing as none of the other parameters in `shellFor` are documented there, I moved it the comment to the `ctl-as-a-dependency` docs alongside the others